driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudHelper.java

```diff
@@ -1182,4 +1197,72 @@ public OperationResult executeEstimatedDocumentCount(final BsonDocument operatio
     private String createRandomEntityId() {
         return "random-entity-id" + uniqueIdGenerator.getAndIncrement();
     }
+
+    /**
+     * The tests in this list can not currently pass when using {@link ChangeStreamDocument} because there is information less when
+     * decoding into an instance of that class.  So for the offending tests, we just decode directly into {@link BsonDocument}.  For all
+     * others, we decode into {@link ChangeStreamDocument} and from there to {@link BsonDocument} so that there are integration tests of
+     * {@link ChangeStreamDocument}.
+     */
+    private static final List<String> OFFENDING_CHANGE_STREAM_TESTS = asList(
```

Comment:
Nit: Possibly change to `BSON_DOCUMENT_CHANGE_STREAM_TESTS` - not sure they are offending me ;)

Reply:
OFFENDING not OFFENSIVE ;)

Reply:
I know but in colloquial English it amounts to the same thing 🇬🇧